Frameworks/UIKit/StarboardXaml/StarboardXaml.cpp

```diff
@@ -165,6 +165,11 @@ void App::OnActivated(IActivatedEventArgs^ args) {
     UIApplicationActivated(args);
 }
 
+void App::OnFileActivated(FileActivatedEventArgs^ args)
```

Comment:
And to provide more context, we updated vsimporter to use App.Xaml.cpp few iterations back so StarboardXaml is not used any more if someone tries to vsimport their iOS project. We still have to maintain both these code paths so we do not have to force existing developer from having to re-import their project

---
In reply to: [82463737](https://github.com/Microsoft/WinObjC/pull/1100#discussion_r82463737) [](ancestors = 82463737)